UM/Scene/ToolHandle.py

```diff
@@ -25,15 +25,17 @@ class ToolHandle(SceneNode.SceneNode):
     ZAxis = 4
     AllAxis = 5
 
-    DisabledColor = Color(0.5, 0.5, 0.5, 1.0)
```

Comment:
I think you have a good point, but looking at the code in SelectionPass it already makes sure that the toolhandle colors don't collide with object (selection-)colors. The only thing that will currently go wrong is if the themer decides it is prettier to let all the axes use the same color.

Having said that, do you have a good idea about how to split selection & display rendering colors? We could give the toolhandle meshes their selectionpass colors, and then in ToolHandle.render somehow detect a "display-render" and override the render color with a themed color. But I don't know how to detect the renderpass from a render() method.